pyvrp/read.py

```diff
@@ -176,18 +184,11 @@ def read(
         )
         for idx in range(dimension)
     ]
-    vehicle_types = [VehicleType(capacity, num_vehicles)]
-
-    if max(distances.max(), durations.max()) > MAX_USER_VALUE:
-        msg = """
-        The maximum distance or duration value is very large. This might
-        impact numerical stability. Consider rescaling your input data.
-        """
-        warn(msg, ScalingWarning)
+    vehicle_types = [VehicleType(num_vehicles, capacity=capacity)]
```

Comment:
If we have multiple depots and we want to use them meaningfully, we also need at least one vehicle type for each depot. However, I'm not sure what the number of vehicles for each type should be... If in VRPLIB format it is assumed the total number of vehicles can be distributed over the different depots but there's no fixed assignment, then this is something we do not currently support... in that case, does it makes sense to instantiate multiple depots at all?

Some ideas:
- support flexible assignment of vehicles but this is a rather large and difficult change (somewhat like a generalized fleetmanager #31 )
- only support fixed num vehicles per depot and add new section to VRPLIB format to provide this information.
- not support multiple depot via VRPLIB format

If we're going to run academic benchmarks we should be careful how to compare results given these differences.